packages/graphql/tests/integration/interface-relationships/declare-relationship/interface-filters.int.test.ts

```diff
@@ -141,13 +141,352 @@ describe("interface filters of declared relationships", () => {
                 episodeNr,
             }
         );
+        await session.close();
     });
 
     afterEach(async () => {
-        await cleanNodesUsingSession(session, [Movie, Series, Actor, Episode]);
+        await cleanNodes(driver, [Movie, Series, Actor, Episode]);
```

Comment:
I think you have still to use cleanNodesUsingSession as the driver does not contain the correct database name and instead will use the default one. There is a Trello card for that!